src/backend/distributed/commands/create_distributed_table.c

```diff
@@ -229,8 +230,15 @@ create_distributed_table(PG_FUNCTION_ARGS)
 
 	char *colocateWithTableName = text_to_cstring(colocateWithTableNameText);
 
+	if (shardCount < 1 || shardCount > 64000)
```

Comment:
should we enforce a maximum? I think how many shards you are able to create is very machine-dependent.

Reply:
These are the limits for `citus.shard_count` (and I also tried to match the error message with that)
I can remove the max.

Reply:
ah, I did not realize these are pre-existing limits